cove_360/templates/cove_360/explore.html

```diff
@@ -210,7 +210,7 @@ <h3 class="panel-title panel-title-explore">
         {% for message, json_location, spreadsheet_location in quality_accuracy_checks %}
         <tr>
             <td>
-              <strong>{{message.heading}}</strong><br>{{message.message}}
+              <strong>{{ message.heading | safe }}</strong><br>{{ message.message | safe }}
```

Comment:
Instead of using the template filter `safe`, I think it's better to use `django.utils.html.mark_safe` in the view ([example](https://github.com/OpenDataServices/cove/blob/28db5ada575aaa81191054ebddfe5962b34f1f39/cove_ocds/lib/exceptions.py#L54)). This is because it brings the declaration of the safety of the string closer to where its being manipulated, which makes it less likely that someone will make it unsafe in future.

Reply:
Thanks @Bjwebb . I agree with you it is much better to do it with mark_safe.